tutorials/plot_configuration.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+========================================
+Background information on configurations
+========================================
+
+This tutorial gives a short introduction to MNE configurations.
+"""
+import os.path as op
+
+import mne
+from mne.datasets.sample import data_path
+
+fname = op.join(data_path(), 'MEG', 'sample', 'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(fname)
+raw.crop(10)
+
+###############################################################################
+# MNE python stores configurations to a folder called `.mne` in the user's
+# home directory, These configurations include information like sample data
+# paths and plotter window sizes. Files inside this folder should never be
+# modified manually. Let's see what the configurations contain.
+print(mne.get_config())
+
+###############################################################################
+# We see fields like "MNE_DATASETS_SAMPLE_PATH". As the name suggests, this is
+# the path the sample data is downloaded to. All the fields in the
+# configuration file can be modified by calling :func:`mne.set_config`.
+# Configurations also include the default logging level for the functions. This
+# field is called "MNE_LOGGING_LEVEL".
+mne.set_config('MNE_LOGGING_LEVEL', 'INFO')
+print(mne.get_config(key='MNE_LOGGING_LEVEL'))
+
+###############################################################################
+# The default value is now set to INFO. This level will now be used by default
+# every time we call a function in MNE. We can set the global logging level for
+# only this session by calling :func:`mne.set_log_level` function.
+mne.set_log_level('WARNING')
+print(mne.get_config(key='MNE_LOGGING_LEVEL'))
```

Comment:
Keep in mind that any dev who builds docs will now have their config changed (bad). It would be better to do something like:
```
original_level = mne.get_config('MNE_LOGGING_LEVEL', 'INFO')
```
and at the end do:
```
mne.set_config('MNE_LOGGING_LEVEL', original_level)
```
There is still some risk to modifying someone's config during build if they stop the build in the middle, but at least that risk is lower than in the current iteration.